config/locales/en.yml

```diff
@@ -324,6 +324,7 @@ en:
       information_icon: Information icon
     nextflow:
       required: Required
+      update_samples: Update Samples with Analysis Results
```

Comment:
**nitpick:** Can you sentence case this, all the rest of the labels on pipelines are that way.

Reply:
Updated in [dc9b1d1](https://github.com/phac-nml/irida-next/pull/544/commits/dc9b1d19d7440b68ff6cdcea281fe9c173fb4ab6)